services/buildFreePizzaEvents.js

```diff
@@ -0,0 +1,20 @@
+const meetupEvent = require('../models/meetupEvent');
+
+module.exports = apiData => {
```

Comment:
I think it would be cool to make a shared `services/buildEvents.js` module, instead of having one for each type of "food" (😂 ), and then have an enum somewhere which is something like:
```
export const FOOD_TYPES = {
  beer: {
    searchTerm: 'free beer'
  },
  pizza: {
    searchTerm: 'free pizza'
  }
};
```
Pass it into `buildEvents.js` as a `searchTerm` param, where it's called, and use it on line 15 here, for example, `return description && description.includes(searchTerm)`.

But I think this could be a further piece of work after this PR, rather than part of it.

Reply:
I like the separation of the services so they both "just do one thing" but I think there's an argument for having one that's shared. What do you think?